primitives/src/targeting.rs

```diff
@@ -102,11 +103,12 @@ pub struct Global {
     pub advertiser_id: String,
     pub country: String,
     pub event_type: String,
-    pub campiagn_id: String,
+    pub campaign_id: String,
     pub campaign_total_spent: String,
     pub campaign_seconds_active: u64,
     pub campaign_seconds_duration: u64,
     pub campaign_budget: BigNum,
+    pub deposit_asset: String,
```

Comment:
this can be removed once we merge #308 either here or in `dev` since the `global.channel` contains also the deposit asset:
https://github.com/AdExNetwork/adex-validator-stack-rust/blob/dev/primitives/src/channel.rs#L83